bundled/tool/lsp_server.py

```diff
@@ -481,6 +487,82 @@ def did_change_configuration(
     """
 
 
+@LSP_SERVER.feature(TEXT_DOCUMENT_DID_OPEN)
+async def did_open(ls: KedroLanguageServer, params: DidOpenTextDocumentParams):
+    await validate_catalog(ls, params.text_document.uri)
+
+
+@LSP_SERVER.feature(TEXT_DOCUMENT_DID_CHANGE)
+async def did_change(ls: KedroLanguageServer, params: DidChangeTextDocumentParams):
+    await validate_catalog(ls, params.text_document.uri)
+
+
+async def validate_catalog(ls: KedroLanguageServer, uri: str):
```

Comment:
It should try to ignore catalog entries that started with `_` as it's not a valid dataset